test/events/git_event.test.ts

```diff
@@ -83,5 +87,9 @@ describe("Test git_event functions", function () {
       const fileChangeContext: any = contexts.find((n: any) => n.schema.includes("file_change"));
       expect(Object.keys(fileChangeContext.data)).to.eql(['file_name', 'insertions','deletions']);
     });
+
+    it("does not encrypt the file_change file_names", async function() {
+
```

Comment:
test?

Reply:
blah!!! I started refactoring and completely forgot 🤦

Reply:
added